src/tree/namespace/hasher.cairo

```diff
@@ -0,0 +1,82 @@
+use alexandria_bytes::Bytes;
+use alexandria_bytes::BytesTrait;
+use blobstream_sn::tree::namespace::merkle_tree::{
+    Namespace, NamespaceNode, NamespaceMerkleMultiproof, NamespaceMerkleProof
+};
+use alexandria_math::U256BitShift;
+use debug::PrintTrait;
+use blobstream_sn::tree::consts::{LEAF_PREFIX, NODE_PREFIX, parity_share_namespace};
+
+fn leaf_digest(namespace: Namespace, data: @Bytes) -> NamespaceNode {
+    // let mut bytes = BytesTrait::new(1, array![LEAF_PREFIX]);
+    let mut bytes = BytesTrait::new_empty();
+    bytes.append_u8(LEAF_PREFIX);
+    // let mut bytes2 = BytesTrait::new_empty();
+    bytes.append_u8(namespace.version);
+    append_bytes31(ref bytes, namespace.id);
+    // bytes.concat(@bytes2);
+    bytes.concat(data);
+    let val = bytes.sha256();
```

Comment:
Computing sha256 hash here and not using resulting value

Reply:
Yep sure